modules/test-utils/src/main/java/org/jboss/wsf/test/AppclientHelper.java

```diff
@@ -193,6 +193,7 @@ private static AppclientProcess newAppclientProcess(final String archive, final
             javaOptsValue.append(acDeubOpts).append(" ");
          }
          pb.environment().put("JAVA_OPTS", javaOptsValue.toString());
+         pb.environment().put("MODULE_OPTS", "--add-modules=java.se");
```

Comment:
A bit of guess as I didn't play with appclient script - shouldn't be `--add-modules=java.se` appended to JAVA_OPTS ?